client/components/error-notice/index.js

```diff
@@ -0,0 +1,26 @@
+/**
+ * External dependencies
+ */
+import React, { PropTypes } from 'react';
+
+/**
+ * Internal dependencies
+ */
+import Notice from 'components/notice';
+
+const ErrorNotice = ( { children, isWarning } ) => {
+	return (
+		<Notice
+			className="error-notice"
+			status={ isWarning ? 'is-warning' : 'is-error' }
+			showDismiss={ false }>
+			{ children }
+		</Notice>
+	);
+};
+
+ErrorNotice.propTypes = {
+	isWarning: PropTypes.boolean,
+};
+
+export default ErrorNotice;
```

Comment:
I see that there are 2 uses of `<ErrorNotice isWarning />`, 1 use of `<ErrorNotice>` and 4 uses of the original `<Notice />` left. Should we either create one WCS-specific notice for all 7 cases, or use the original `<Notice />` in all cases?